flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -78,6 +82,7 @@ import org.apache.flink.util.{InstantiationUtil, NetUtils}
 import org.jboss.netty.channel.ChannelException
 
 import scala.annotation.tailrec
+import scala.collection.JavaConversions._
```

Comment:
I'll change it in the JM. Since other lines in the TM may rely on that import this should imo be handled in another PR.